cmd/dnn-ssd/main.go

```diff
@@ -0,0 +1,140 @@
+// What it does:
+//
+// This example shows, how to use pretrained SSD (Single Shot Detection) detection networks in gocv.
+// here, we detect human faces from the camera, but the setup is similar for any other kind of object detection.
+//
+// Download the Caffe model file from:
+// https://raw.githubusercontent.com/opencv/opencv_3rdparty/dnn_samples_face_detector_20180205_fp16/res10_300x300_ssd_iter_140000_fp16.caffemodel
+// (no fear, this one is fairly small, 5.1mb)
+//
+// Also, you will need the prototxt file:
+// https://raw.githubusercontent.com/opencv/opencv/master/samples/dnn/face_detector/deploy.prototxt
+//
+// How to run:
+//
+// 		go run ./cmd/dnn-ssd/main.go 0 [protofile] [modelfile]
+//
+// +build example
+
+package main
+
+import (
+	"fmt"
+	"image"
+	"image/color"
+	"os"
+	"strconv"
+
+	"gocv.io/x/gocv"
+)
+
+func min(a, b float32) float32 {
```

Comment:
Is it because you use `float32` that you cant use [`math.Max`](https://golang.org/pkg/math/#Max)?

Reply:
yes, indeed. go offers only Min, Max for float64

Reply:
Should we add a "utils" lib in gocv? These `min`/`max` functions are pretty useful.

Reply:
maybe even join that with the test_helper.go, which has round()

Reply:
right, although I was thinking about exposing these functions in userland

Reply:
more like: rename helpers_test.go to utility.go, and make it public

Reply:
on the other hand, we have so much under the `gocv` "package" already that maybe we should thing about a sub-package and also we need to be smart because we likely need these functions for `int`, `float32`, etc.

Maybe we could gather the functions we use all the time when working with opencv that dont exist natively in Go, and create a sub-package: `gocv.math` with `MinInt`, `MinInt64`, `MinFloat32`, etc. WDYT? Just thinking it out loud..

Reply:
yea, the types are a problem, and that would solve it, imho.